src/app/Library/CrudPanel/Traits/Read.php

```diff
@@ -143,26 +143,28 @@ public function enableBulkActions()
             'type'            => 'checkbox',
             'name'            => 'bulk_actions',
             'label'           => ' <input type="checkbox" class="crud_bulk_actions_main_checkbox" style="width: 16px; height: 16px;" />',
-            'priority'        => 1,
+            'priority'        => 0,
             'searchLogic'     => false,
             'orderable'       => false,
             'visibleInTable'  => true,
             'visibleInModal'  => false,
             'visibleInExport' => false,
             'visibleInShow'   => false,
+            'hasActions'      => true,
         ])->makeFirstColumn();
 
         $this->addColumn([
             'type'            => 'custom_html',
             'name'            => 'blank_first_column',
             'label'           => ' ',
-            'priority'        => 1,
+            'priority'        => 0,
             'searchLogic'     => false,
             'orderable'       => false,
             'visibleInTabel'  => true,
             'visibleInModal'  => false,
             'visibleInExport' => false,
             'visibleInShow'   => false,
+            'hasActions'      => true,
```

Comment:
By adding a `hasActions` flag to columns, we can easily count how many _real_ columns there are, and exclude the columns that have actions.